libs/books/feature/src/lib/book-search/book-search.component.html

```diff
@@ -1,49 +1,22 @@
 <form [formGroup]="searchForm" (submit)="searchBooks()">
-  <mat-form-field floatLabel="never">
+  <mat-form-field>
+    <mat-label id="search-label">Search</mat-label>
     <input
+      aria-labelledby="search-label"
```

Comment:
You don't actually need to do this. Angular Material should take care of this for you.